models/issue.go

```diff
@@ -1156,17 +1157,20 @@ func (opts *IssuesOptions) setupSession(sess *xorm.Session) {
 
 	if opts.ProjectID > 0 {
 		sess.Join("INNER", "project_issue", "issue.id = project_issue.issue_id").
-			And("project_issue.project_id=?", opts.ProjectID)
+			And("project_issue.project_id=?", opts.ProjectID).OrderBy("`project_issue`.priority")
 	}
-
 	if opts.ProjectBoardID != 0 {
 		if opts.ProjectBoardID > 0 {
-			sess.In("issue.id", builder.Select("issue_id").From("project_issue").Where(builder.Eq{"project_board_id": opts.ProjectBoardID}))
+			sess.In("issue.id", builder.Select("issue_id").From("project_issue").Where(builder.Eq{"project_board_id": opts.ProjectBoardID}).OrderBy("priority"))
```

Comment:
imho it would be safer to use 
```
`issue`.priority
```
in order by

Reply:
fixed